### PR TITLE
Myth: ARIA Has Perfect Support

### PR DESCRIPTION
**Pull request by ericwbailey**
This PR adds a new post about the complexities behind ARIA support considerations.

### CODE REVIEW

File: _posts/2020-05-13-aria-has-perfect-support.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+``` html
+<button type="button">
+  <svg aria-hidden="true" focusable="false">
+    <!-- SVG code -->
+  </svg>
+  Save
+</button>
+```
+
+ARIA can also be used for interactive controls such as accordions, tabs, and modals. A thoughtful, skilled application of ARIA can help communicate things like if an accordion is in an expanded state, which tab is currently being viewed, or what is and is not modal content.
```
Comment (scottaohara) [inline _posts/2020-05-13-aria-has-perfect-support.md:42]: an accordion panel...

...which tab's associated tab panel is currently visible...

File: _posts/2020-05-13-aria-has-perfect-support.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Nearly all ARIA is inserted and updated dynamically via JavaScript. Very little of it is “hard coded” into the HTML it is manipulating. There are many ways to write this JavaScript. You can use “vanilla” JavaScript, or specialized frameworks such as [Ember](https://emberjs.com/), [Angular](https://angular.io/), [React](https://reactjs.org/), [Vue](https://vuejs.org/), [Svelte](https://svelte.dev/), etc. ARIA is compatible with all these frameworks because [it is a standard](https://w3c.github.io/aria/) they observe.
```
Comment (scottaohara) [inline _posts/2020-05-13-aria-has-perfect-support.md:44]: "Nearly all ARIA is inserted and updated dynamically via JavaScript."

should "is inserted" be "should be inserted"???

File: _posts/2020-05-13-aria-has-perfect-support.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- The version of the operating system,
+- The kind of browser running on the operating system,
+- The version of browser running on the operating system,
+- Any modification a person has made to any of the preceding. 
+
+Compatibility issues means there might be discrepancies in how web technology is understood and shown to the person using it. In many cases, it won’t be shown at all.
+
+ARIA is a web technology. In addition to the issues listed above, it can also have compatibility issues that come from:
+
+- The kind of assistive technology being used
```
Comment (scottaohara) [inline _posts/2020-05-13-aria-has-perfect-support.md:60]: previous list uses commas at end.  do that here?

File: _posts/2020-05-13-aria-has-perfect-support.md (hunk 2 above)
Comment (ericwbailey) [reply _posts/2020-05-13-aria-has-perfect-support.md:44]: Ideally 😅

File: _posts/2020-05-13-aria-has-perfect-support.md (hunk 3 above)
Comment (ericwbailey) [reply _posts/2020-05-13-aria-has-perfect-support.md:60]: Done.

**Comment by ericwbailey** [comment]
Okey doke. Pushed up changes.

**Pull request merged**